src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  const num1 = new Big(numberOne);
+  const num2 = new Big(numberTwo);
+  if (operation === '-') {
+    result = num1.minus(num2);
+  }
+  if (operation === '+') {
+    result = num1.plus(num2);
+  }
+  if (operation === 'X') {
+    result = num1.times(num2);
+  }
+  if (operation === '÷') {
```

Comment:
- [ ] Let's add another case for [`modifier`](https://mikemcl.github.io/big.js/#mod)